### DISCUSSION TITLE
Release/5.3.2

### DISCUSSION THREAD

**Discussion by Athanasius** [EDCD/EDMarketConnector#1502] [Announcement]
Release 5.3.2
===

This release contains one change to cope with how Frontier decided to name the Journal files differently in the Update 11 Odyssey client.

<hr /><em>This discussion was created from the release <a href='https://github.com/EDCD/EDMarketConnector/releases/tag/Release/5.3.2'>Release/5.3.2</a>.</em>

**Comment by judgechuck** [comment]
Hi, firstly, thanks for a great tool! :-)

I have upgraded ED to Odyssey Update 11 and EDMC wasn't detecting my Commander. On seeing that you have already updated EDMC because of the FDev changes, I downloaded and installed Release 5.3.2, however, I am still seeing the same issue, "Status: Don't yet know your Commander name".
I have tried quitting ED and EDMC and re-loading, but still having the same issue. Any ideas?

Regards,

CMDR Judge Chuck

**Reply by Athanasius** [reply]
That shouldn't be happening.  I tested 5.3.2, **as released**, myself with first Odyssey and then Horizon clients.  It worked as expected.

Double-check you're actually getting Journal files in `shell:SavedGames\Frontier Developments\Elite Dangerous\`

**Reply by Athanasius** [reply]
Oh, also, you say 'quitting ED ... and re-loading'.  That **did** include going **fully in-game**, didn't it ?  The Journal only writes the event that states your Commander name at that point.

**Reply by judgechuck** [reply]
Hi Athanasius, yes, definitely. Checked in EDMC settings and ED Journal file location is set to:
D:\Users\username\Saved Games\Frontier Developments\Elite Dangerous
I have also confirmed that journal files are located there and being updated. Here is the header (with some redacted data) from the latest log:
{ "timestamp":"2022-03-15T14:28:29Z", "event":"Fileheader", "part":1, "language":"English/UK", "Odyssey":true, "gameversion":"4.0.0.1200", "build":"r282108/r0 " }
{ "timestamp":"2022-03-15T14:29:11Z", "event":"Commander", "FID":"F1234567", "Name":"CMDRName" }

Finally, yes, I have fully quit out of both applications (ED & EDMC) and re-loaded, including fully logging in. Are there any EDMC logs that I can send you?

Edited to add:
There is a file being created in my save folder "edmc-journal-lock.txt", contents are:
Path: D:\Users\username\Saved Games\Frontier Developments\Elite Dangerous
PID: 42168

Second edit: I am also using EDDiscovery and that is still working fine, as far as I can see.

Regards,

JC

**Reply by judgechuck** [reply]
OK, no idea what I've done but fourth or fifth attempt seems to be the charm. I quit right out of Steam and re-loaded from scratch and it looks good now. I may have not quit out of the ED "loader" app initially. Sorry for the bother and thanks for the quick reply!

Regards, JC

**Reply by Athanasius** [reply]
Actually I've just reproduced this behaviour when EDMC is started **after** the game.  It's picking up the wrong journal file, if there were some old-name-style ones there.

I'll investigate, hopefully find a fix, and release a 5.3.3 later today.

**Reply by jaydeeveloper** [reply]
I'm facing the same issue, can't find where are the app logs.

**Reply by judgechuck** [reply]
Ah, excellent, so I'm not completely bonkers! ;-) Will look forward to another update but, as long as I'm starting things in order, everything is working perfectly.

Regards,

JC

**Comment by Athanasius** [comment]
[5.3.3](https://github.com/EDCD/EDMarketConnector/releases/tag/Release%2F5.3.3) is now out and should have fully addressed the issues with the Journal file names.
